settings.gradle.kts

```diff
@@ -67,6 +67,9 @@ project(":arrow-core-serialization").projectDir = file("arrow-libs/core/arrow-co
 include("arrow-cache4k")
 project(":arrow-cache4k").projectDir = file("arrow-libs/core/arrow-cache4k")
 
+include("arrow-autocloseable")
```

Comment:
Nitpick: I think it's better to call the library `arrow-autoclose`, since that's the name of the scope